### PR TITLE
kcli: init at 1.8.2

### PR DESCRIPTION
**Pull request by cswank**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/kcli/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    rev = version;
+    sha256 = "1m9967f9wk1113ap2qmqinqg7gvpmg5y2g1ji0q818qbandzlh23";
+  };
+
+  modSha256 = "1wcqh3306q9wxb6pnl8cpk73vmy36bjv2gil03j7j4pajs1f2lwn";
+
+  subPackages = [ "." ];
+
+  meta = with stdenv.lib; {
+    description = "A kafka command line browser.";
```
Comment (cab404) [inline pkgs/development/tools/kcli/default.nix]: dot at the end should be omited iirc

Comment (cswank) [reply pkgs/development/tools/kcli/default.nix]: fixed

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -1471,6 +1471,12 @@
     githubId = 143982;
     name = "Charles Strahan";
   };
+  cswank = {
```
Comment (kalbasit) [inline maintainers/maintainer-list.nix]: Please put this change in a separate commit than the kcli init.

File: pkgs/development/tools/kcli/default.nix [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  src = fetchFromGitHub {
+    owner = "cswank";
+    repo = "kcli";
+    rev = version;
+    sha256 = "1m9967f9wk1113ap2qmqinqg7gvpmg5y2g1ji0q818qbandzlh23";
+  };
+
+  modSha256 = "1wcqh3306q9wxb6pnl8cpk73vmy36bjv2gil03j7j4pajs1f2lwn";
+
+  subPackages = [ "." ];
```
Comment (kalbasit) [inline pkgs/development/tools/kcli/default.nix]: This can probably be removed unless you're getting errors without it.

File: pkgs/development/tools/kcli/default.nix [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+  meta = with stdenv.lib; {
+    description = "A kafka command line browser";
+    homepage = https://github.com/cswank/kcli;
```
Comment (kalbasit) [inline pkgs/development/tools/kcli/default.nix]: Please put the URL in quotes.

**Review by kalbasit — CHANGES_REQUESTED** [review]

File: pkgs/development/tools/kcli/default.nix (hunk 3 above)
Comment (cswank) [reply pkgs/development/tools/kcli/default.nix]: I am indeed getting errors without it.

File: maintainers/maintainer-list.nix (hunk 2 above)
Comment (cswank) [reply maintainers/maintainer-list.nix]: done

File: pkgs/development/tools/kcli/default.nix (hunk 4 above)
Comment (cswank) [reply pkgs/development/tools/kcli/default.nix]: done

**Review by kalbasit — APPROVED** [review]

**Comment by kalbasit** [comment]
@GrahamcOfBorg build kcli
